Remotion/ObjectBinding/Web/UI/Controls/BocListImplementation/Rendering/BocSelectorColumnRenderer.cs

```diff
@@ -168,6 +175,9 @@ private void RenderDataRowSelectorControl (BocListRenderingContext renderingCont
 
       renderingContext.Writer.RenderBeginTag (HtmlTextWriterTag.Input);
       renderingContext.Writer.RenderEndTag();
+      renderingContext.Writer.AddAttribute (HtmlTextWriterAttribute.Class, "checkbox-visualizer");
```

Comment:
I don't think we need to add a css-class here and before. We should be able to do this with the `remotion-themed` css class on the input element  and moving all the css and svg content into remotion.web. The only thing we need to do in objectbinding should be adding the css-class and the empry span (and click handler, apparently).